src/server/api/routes/users.router.js

```diff
@@ -58,4 +58,54 @@ router.get('/:id', (req, res, next) => {
     .catch(next);
 });
 
+/**
+ * @swagger
+ * /modules:
+ *  post:
+ *    summary: Create a user
+ *    description:
+ *      Will create a user.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: user
+ *        description: The module to create a user.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - title
+ *            - startDate
+ *            - endDate
+ *             - classId
+ *          properties:
+ *            title:
+ *              type: string
+ *            startDate:
+ *              type: string
+ *              format: date-time
+ *            endDate:
+ *              type: string
+ *              format: date-time
+ *            classId:
+ *              type: string
```

Comment:
this schema doesn´t look correct to me. 
You can [see the schema here](https://dbdiagram.io/d/5ffc249780d742080a35cb4a).

Reply:
Thanks Ember,.let me correct it,